src/ontology/mondo-ingest.Makefile

```diff
@@ -486,18 +486,10 @@ slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/%_map
 slurp-%: slurp/%.tsv
 	@echo "$@ completed".
 
-.PHONY: slurp-no-updates-%
-slurp-no-updates-%: slurp/%.tsv
-	@echo "$@ completed".
-
 .PHONY: slurp-docs
 slurp-docs:
 	python3 $(SCRIPTSDIR)/migrate.py --docs
 
-.PHONY: slurp-all-no-updates
-slurp-all-no-updates: $(foreach n,$(ALL_COMPONENT_IDS), slurp-no-updates-$(n))
-	@echo "$@ ($^) completed".
```

Comment:
## Skip build QC for this one?
This seems like a good case of skipping the need to run a build, because nowhere in `build-mondo-ingest` are these goals called. Deletion of them should have no effect on the build.

Reply:
In this very specific and limited case, it's ok to not run a test build for this change.